memgpt/agent.py

```diff
@@ -582,7 +580,12 @@ def step(self, user_message, first_message=False, first_message_retry_limit=FIRS
                 printd(f"This is the first message. Running extra verifier on AI response.")
                 counter = 0
                 while True:
-                    response = get_ai_reply(model=self.model, message_sequence=input_message_sequence, functions=self.functions)
+                    response = get_ai_reply(
+                        model=self.model,
+                        message_sequence=input_message_sequence,
+                        functions=self.functions,
+                        context_window=self.config.context_window,

```

Comment:
@sarahwooders example of grabbing the config inside of the `Agent` instance, is this correct?

Reply:
`[agent/self].config.context_window`